rundeckapp/grails-app/views/user/login.gsp

```diff
@@ -58,7 +58,7 @@
         <div class="container">
           <div class="row">
             <div class="col-md-4 col-sm-6 col-md-offset-4 col-sm-offset-3">
-              <form action="j_security_check" method="post" class="form " role="form">
+              <form action="/j_security_check" method="post" class="form " role="form">
```

Comment:
I think if rundeck is deployed into a context such as `/rundeck` (e.g. in tomcat) this will no longer work...